index.html

```diff
@@ -6,8 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
     <title>Portfolio Skeleton</title>
+    <script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    
+    <header>
+        <nav>
+            <ul>
+                <li><img src="./Normal Button/Tertiary/Icons/MyLogo.png" alt="My-Logo"></li>
+                <li><img src="./Normal Button/Tertiary/Icons/Icon - Menu.png" alt="Icon-Menu"></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="headline">
+            <h1>Greetings.<br> I'm Manuel.</h1>
+            <h3>I'll become a Software Developer.</h3>
+            <p>I'm doing my best to improve my habilities in building responsive web design projects. Any comment of this template is welcome.</p>
+            <div class="icons">
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-twitter"></i>
+            </div>
```

Comment:
- [ ] Please I recommend wrapping the icons with `ul` tags before `li` tags and adding `a` tags to each `li` with links per the project requirements. 😄